Updated Character Selection UI

TNWing opened the pull request:
Character Selection now displays Vash and Octavia as the two choices (only Vash is playable currently). Furthermore, their icons are displayed to the right of their name

TNWing committed:
Updated Character Selection UI

Character Selection now displays Vash and Octavia as the two choices (only Vash is playable currently).
Furthermore, their icons are displayed to the right of their name

TNWing committed:
game ui size adjustment

made game ui slightly smaller + started to prep other ui for more clear selections

TNWing committed:
basic setup for instructions window in main menu

adding to the more clear selection, an empty, untested instruction window is now available in the main menu

TNWing committed:
minor main menu adjustment

just setting some variables to prevent bugs

njpires20 reviewed (changes requested):
Needs changes. See discord

TNWing committed:
ui more clear

the ui should not screw up the colors of the butotn when changing selection now

TNWing committed:
placeholder instruction image + cleanup

instruction button now has use. to exit instructions, press the bottom button on gamepad (generally the A button on Xbox controllers)
